_i18n/en/documentation/general/other-platforms.md

```diff
@@ -0,0 +1,12 @@
+However attractive the title may sound, AntennaPod is not and will not be available on Windows, iOS or the Linux desktop. The app is made by a small team of volunteers, and making the app for these other platforms would be a lot of work.
+
+There are (open source) alternatives, however, which we would like to point you to:
+App/Application|Windows|iOS|Linux
+---|---|---|---
+[Kasts](https://apps.kde.org/kasts/)|No|No|Yes
+[Vocal](https://vocalproject.net/)|No|No|Yes
+[gPodder](https://gpodder.github.io/)|Yes|Yes|Yes
```

Comment:
Gpodder runs on iOS?

Reply:
Hmm. Good point. Maybe we should split them up.